### PR TITLE
Suhas local

### PR DESCRIPTION
**Pull request by SuhasVMM**

### CODE REVIEW

File: src/test/java/org/shikshalokam/backend/ep/ElevateProjectBaseTest.java [hunk 1]
```diff
@@ -19,8 +21,10 @@ public class ElevateProjectBaseTest extends MentorBase {
     public static String User_ID = null;
     public static String BASE_URL = PropertyLoader.PROP_LIST.getProperty("elevate.qa.api.base.url");
     public static String INTERNAL_ACCESS_TOKEN = PropertyLoader.PROP_LIST.getProperty("elevate.internalaccesstoken");
+    public static String CRUDEntityType_Name = PropertyLoader.PROP_LIST.getProperty("elevate.qa.CRUDEntitytype.name");
```
Comment (infa-rvarahag) [inline src/test/java/org/shikshalokam/backend/ep/ElevateProjectBaseTest.java]: remove this

File: src/test/java/org/shikshalokam/backend/ep/ElevateProjectBaseTest.java [hunk 2]
```diff
@@ -19,8 +21,10 @@ public class ElevateProjectBaseTest extends MentorBase {
...
+    public static String entityType_Id;
```
Comment (infa-rvarahag) [inline src/test/java/org/shikshalokam/backend/ep/ElevateProjectBaseTest.java]: remove this

File: src/test/java/org/shikshalokam/backend/ep/ElevateProjectBaseTest.java [hunk 3]
```diff
@@ -35,5 +39,42 @@ public static Response loginToElevate(String email, String Password) {
...
+        requestBody.put("projection", projection);
+        requestBody.put("skipFields", skipFields);
+
+        Response response = given()
+                .header("Authorization", INTERNAL_ACCESS_TOKEN)
+                .header("internal-access-token", INTERNAL_ACCESS_TOKEN)
+                .header("x-auth-token", X_AUTH_TOKEN)
+                .header("Content-Type", "application/json")
+                .body(requestBody) // Send the HashMap request body
+                .post(BASE_URL + PropertyLoader.PROP_LIST.getProperty("elevate.qa.fetchentitytype.endpoint"));
```
Comment (infa-rvarahag) [inline src/test/java/org/shikshalokam/backend/ep/ElevateProjectBaseTest.java:64]: Not needed below code of if statement

Comment (infa-rvarahag) [inline src/test/java/org/shikshalokam/backend/ep/ElevateProjectBaseTest.java:64]: No need to append Base url

**Review by infa-rvarahag — CHANGES_REQUESTED** [review]
Mainly remove the not needed headers

**Review by infa-rvarahag — CHANGES_REQUESTED** [review]
Submit only single entity code

**Comment by SuhasVMM** [comment]
Raising new PR

**Pull request closed without merging**